langlib/java/src/main/ballerina/ballerina_values_to_Java.bal

```diff
@@ -91,6 +91,18 @@ public isolated function getClass(string name) returns handle | error = @Method
     paramTypes: ["io.ballerina.runtime.api.values.BString"]
 } external;
 
+# Performs a direct conversion of a value to a string.
+# The conversion is direct in the sense that when applied to a value that is already
+# a string it leaves the value unchanged.
+#
+# + v - the value to be converted to a string
+# + return - a string resulting from the conversion
+public isolated function getStringValue((any) v) returns string = @Method {
```

Comment:
Is there a reason why this is a `public` function? And also if this is only used in StackFrameImpl.toString method shall we make this to take `int` parameters.

Reply:
Fixed in c87668a